docs/product-user-guide/src/main/asciidoc/guided-rules-THEN-proc.adoc

```diff
@@ -1,31 +1,45 @@
-[id='guided_rules_THEN_proc']
-= Adding THEN Actions in Guided Rules
+[id='guided-rules-THEN-proc']
+= Adding THEN actions in guided rules
 
 The *THEN* part of the rule is the action to be performed when the *WHEN* condition of the rule has been met. For example, when a loan applicant is under 21 years old, the *THEN* action might set `approved` to `false`, declining the loan because the applicant is under age. You can set simple or complex actions to determine how and when your rules are applied.
 
 .Prerequisite
-All data objects required for your rules are listed in the *Data Objects* tab of the Guided Rule Editor. See <<guided_rules_create_proc>> for details.
+All data objects required for your rules have been created or imported and are listed in the *Data Objects* tab of the guided rules designer.
 
 .Procedure
-. In the Guided Rule Editor, click the plus icon (image:5686.png[]) on the right side of the `THEN` section.
+. In the guided rule designer, click the plus icon (image:5686.png[]) on the right side of the `THEN` section.
 +
-The *Add a new action...* dialog window with the available action templates opens.
+The *Add a new action* window with the available action elements opens.
 +
-.Nurse Roster THEN Dialog Window
-image::5695.png[]
-. Choose an action (for example *Modify a ...*) and click *Ok*.
-. Click the newly added `THEN` action.
+.Add a new action to the rule
+image::5695.png[Add a new action to the rule]
 +
-An *Add a field* dialog appears. In this example, the field is for the "*Modify value of LoanApplication [a]*" image:6191.png[] action.
+The list includes insertion and modification options based on the data objects in the *Data Objects* tab of the guided rule designer, and on any DSL objects defined for the package (if you selected *Use Domain Specific Language (DSL)* when you created this guided rule):
+
+* *Change field values of:* Use this to set the value of fields on a fact (such as `LoanApplication`).
```

Comment:
IDK if you want to differentiate a little more between "Change" and "Modify" as they seem to have the same function in this documentation. A "Change" does not, by default, inform the decision engine that the **Fact** has changed - which could lead to other rules being matched. There is an option to "Update engine" which will inform the decision engine that the whole Fact needs re-evaluating by all rules. A "Modify" automatically informs the decision engine that a **Field** (or Fields) has/have changed (not the whole Fact). The behaviour of "Modify" is more subtle than "Change" (with "Update engine" checked).